runtime-modules/referendum/src/weights.rs

```diff
@@ -140,7 +140,8 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	}
 	// Storage: Instance1Referendum AccountsOptedOut (r:0 w:1)
 	fn opt_out_of_voting() -> Weight {
-		(16_920_000 as Weight)
```

Comment:
We do check for duplicates in `opt-out` extr., but that read seems not to be accounted for.

Reply:
Its a shame the comments are not very accurate, but notice the added db read weight on the following lines. Originally the placeholder benchmarks was added in first iteration of [implementation](https://github.com/Joystream/joystream/pull/4409) (didn't have a read), after the suggested fix, the weight generation wasn't run. So it showed up now when  I did the final generation.